galleries/examples/shapes_and_collections/collections.py

```diff
@@ -47,53 +47,36 @@
 
 
 col = collections.LineCollection(
-    [spiral], offsets=xyo, offset_transform=ax1.transData)
+    [spiral], offsets=xyo, offset_transform=ax1.transData, color=colors)
+# transform the line segments such that their size is given in points
 trans = fig.dpi_scale_trans + transforms.Affine2D().scale(1.0/72.0)
 col.set_transform(trans)  # the points to pixels transform
-# Note: the first argument to the collection initializer
-# must be a list of sequences of (x, y) tuples; we have only
-# one sequence, but we still have to put it in a list.
-ax1.add_collection(col, autolim=True)
-# autolim=True enables autoscaling.  For collections with
-# offsets like this, it is neither efficient nor accurate,
-# but it is good enough to generate a plot that you can use
-# as a starting point.  If you know beforehand the range of
-# x and y that you want to show, it is better to set them
-# explicitly, set the *autolim* keyword argument to False.
```

Comment:
This is unclear: Why is it neither efficient nor accurate. I'm also not clear whether the statement is still correct. It's from 2006 and collection limit handling as improved since then. If anything we'd need an explicit discussion on limit handling for collections, but that's not the scope of this example. So let's remove this here.